### ISSUE TITLE
universal build release for c++ library

### ISSUE THREAD

**Issue by gianmarcohutter** [google/sentencepiece#892]
Was wondering if there is an universal release of the libsentencepiece.dylib that work for both arm64 and x84 macs? [sentencepiece-0.1.99-Darwin.7z]([sentencepiece-0.1.99-Darwin.7z](https://github.com/google/sentencepiece/releases/download/v0.1.99/sentencepiece-0.1.99-Darwin.7z)) works great on my x84 machine but I would like to distribute it to work on M1 machines as well. Any help is very appreciated!

**Comment by h-vetinari** [comment]
It's not a multi-architecture build, but there are builds of `libsentencepiece` for `osx-arm64` (i.e. M1/M2/etc.) in conda-forge. Can be installed like:
```
conda install -c conda-forge libsentencepiece  # or mamba install ...
```

**Comment by taku910** [comment]
sentencepiece is built with cmake, so you can simply run the cmake command with the following env to make universal binary. 

env CMAKE_OSX_ARCHITECTURES="x86_64;arm64" cmake ..

**Comment by taku910** [comment]
Will fix it in the next release.

**Comment by taku910** [comment]
Fixed in v0.2.0
